### DISCUSSION TITLE
Install Linux on Mac M1 macMini - only blinking cursor

### DISCUSSION THREAD

**Discussion by tardar** [utmapp/UTM#4812] [UTM Help (macOS)] [answered]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
I try to install kali linux arm with utm on my mac mini.
When i boot the first time the boot menu of the iso will appear, but if i hit "Install" from the grub menue, i only get this:

<img width="800" alt="linux_install_utm_macos" src="https://user-images.githubusercontent.com/40645000/206861251-4b225154-72e7-4adc-9c8b-997443948acb.png">

So the install won't start and it makes no difference if i try to eject the iso or not.

**Configuration**  
* UTM Version: 4.0.9
* macOS Version: 13.0.1
* Mac Chip (Intel, M1, ...): M1 2021

**Debug log**  
The button won't become clickable for me :(

**Upload VM**  
[config.plist.zip](https://github.com/utmapp/UTM/files/10200316/config.plist.zip)

**Comment by adasiko** [comment, marked as answer]
https://docs.getutm.app/guides/kali/#black-screen-on-start

**Comment by bendavis78** [comment]
I'm having the same issue w/ the latest version of UTM and Kali for Applie Silicon, even when trying to install in graphics mode. It just gets stuck on the blinking cursor.
